lib/percy/capybara/client/user_agent.rb

```diff
@@ -0,0 +1,37 @@
+module Percy
+  module Capybara
+    class Client
+      module UserAgent
+        def _client_info
+          "percy-capybara/#{VERSION}"
+        end
+
+        def _environment_info
+          [
+            "percy-capybara-loader/#{loader}",
+            "rails/#{_rails_version}",
+            "sinatra/#{_sinatra_version}",
+            "ember-cli-rails/#{_ember_cli_rails_version}",
+          ].reject do |info|
+            info =~ %r{\/$} # reject if version is empty
```

Comment:
There's a good reason why things are setup this way. The short answer is because it's easier to test.